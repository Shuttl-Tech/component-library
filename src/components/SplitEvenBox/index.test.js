import React from 'react';
import renderer from 'react-test-renderer';

import { SplitEvenBox } from './';

const segmentText1 = 'Micheal Faraday';
const segmentText2 = 'Tim Berners-Lee';
const segmentText3 = 'Marie Curie';
const segmentText4 = 'Louis Pasteur';

it('SplitEvenBox tree looks like how it should', () => {
	let props = {
		segments: [
			<span>{segmentText1}</span>, <span>{segmentText2}</span>, <span>{segmentText3}</span>, <span>{segmentText4}</span>
		]
	};

	const component = renderer.create(<SplitEvenBox {...props} />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
