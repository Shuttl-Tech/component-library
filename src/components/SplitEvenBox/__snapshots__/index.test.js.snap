// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SplitEvenBox tree looks like how it should 1`] = `
<div
  className="box component--box container horizontal component--split-even-box--container component--split-even-box--container--horizontal"
>
  <div
    className="segment component--split-even-box--segment"
  >
    <span>
      Micheal Faraday
    </span>
  </div>
  <div
    className="segment component--split-even-box--segment"
  >
    <span>
      Tim Berners-Lee
    </span>
  </div>
  <div
    className="segment component--split-even-box--segment"
  >
    <span>
      Marie Curie
    </span>
  </div>
  <div
    className="segment component--split-even-box--segment"
  >
    <span>
      Louis Pasteur
    </span>
  </div>
</div>
`;
